src/webui/www/private/scripts/client.js

```diff
@@ -55,6 +57,16 @@ let tagList = {};
 let selectedTag = TAGS_ALL;
 let setTagFilter = function() {};
 
+/* Trackers filter */
+const TRACKERS_ALL = 1;
+const TRACKERS_TRACKERLESS = 2;
+
+const trackerList = {};
```

Comment:
do you mind using `new Map()` here?

Reply:
I don't mind, you are right. We probably should switch over.

Reply:
Updated